recipe/session/sessionwithjwt/sessionContainer.go

```diff
@@ -70,80 +69,42 @@ func newSessionWithJWTContainer(originalSessionClass sessmodels.SessionContainer
-		GetSessionData:                   originalSessionClass.GetSessionData,
-		UpdateSessionData:                originalSessionClass.UpdateSessionData,
-		GetUserID:                        originalSessionClass.GetUserID,
-		GetAccessTokenPayload:            originalSessionClass.GetAccessTokenPayload,
-		GetHandle:                        originalSessionClass.GetHandle,
-		GetAccessToken:                   originalSessionClass.GetAccessToken,
-		GetTimeCreated:                   originalSessionClass.GetTimeCreated,
-		GetExpiry:                        originalSessionClass.GetExpiry,
+		RevokeSessionWithContext:               originalSessionClass.RevokeSessionWithContext,
+		GetSessionDataWithContext:              originalSessionClass.GetSessionDataWithContext,
+		UpdateSessionDataWithContext:           originalSessionClass.UpdateSessionDataWithContext,
+		GetUserIDWithContext:                   originalSessionClass.GetUserIDWithContext,
+		GetAccessTokenPayloadWithContext:       originalSessionClass.GetAccessTokenPayloadWithContext,
+		GetHandleWithContext:                   originalSessionClass.GetHandleWithContext,
+		GetAccessTokenWithContext:              originalSessionClass.GetAccessTokenWithContext,
+		GetTimeCreatedWithContext:              originalSessionClass.GetTimeCreatedWithContext,
+		GetExpiryWithContext:                   originalSessionClass.GetExpiryWithContext,
+		AssertClaimsWithContext:                originalSessionClass.AssertClaimsWithContext,
+		FetchAndSetClaimWithContext:            originalSessionClass.FetchAndSetClaimWithContext,
+		SetClaimValueWithContext:               originalSessionClass.SetClaimValueWithContext,
+		GetClaimValueWithContext:               originalSessionClass.GetClaimValueWithContext,
+		RemoveClaimWithContext:                 originalSessionClass.RemoveClaimWithContext,
+		MergeIntoAccessTokenPayloadWithContext: originalSessionClass.MergeIntoAccessTokenPayloadWithContext,
+
+		RevokeSession:               originalSessionClass.RevokeSession,
+		GetSessionData:              originalSessionClass.GetSessionData,
+		UpdateSessionData:           originalSessionClass.UpdateSessionData,
+		GetUserID:                   originalSessionClass.GetUserID,
+		GetAccessTokenPayload:       originalSessionClass.GetAccessTokenPayload,
+		GetHandle:                   originalSessionClass.GetHandle,
+		GetAccessToken:              originalSessionClass.GetAccessToken,
+		GetTimeCreated:              originalSessionClass.GetTimeCreated,
+		GetExpiry:                   originalSessionClass.GetExpiry,
+		AssertClaims:                originalSessionClass.AssertClaims,
+		FetchAndSetClaim:            originalSessionClass.FetchAndSetClaim,
+		SetClaimValue:               originalSessionClass.SetClaimValue,
+		GetClaimValue:               originalSessionClass.GetClaimValue,
+		RemoveClaim:                 originalSessionClass.RemoveClaim,
+		MergeIntoAccessTokenPayload: originalSessionClass.MergeIntoAccessTokenPayload,
```

Comment:
some of these functions call the updateAccessTokenPayload function - so that calls the jwtAware version of that function right? Have you tested this properly?